ini/features.ini

```diff
@@ -20,7 +20,7 @@ MARLIN_TEST_BUILD                      = build_src_filter=+<src/tests>
 POSTMORTEM_DEBUGGING                   = build_src_filter=+<src/HAL/shared/cpu_exception> +<src/HAL/shared/backtrace>
                                          build_flags=-funwind-tables
 MKS_WIFI_MODULE                        = QRCode=https://github.com/makerbase-mks/QRCode/archive/261c5a696a.zip
-HAS_TRINAMIC_CONFIG                    = TMCStepper=https://github.com/MarlinFirmware/TMCStepper/archive/marlin-2.1.3.x.zip
+HAS_TRINAMIC_CONFIG                    = TMCStepper=https://github.com/MarlinFirmware/TMCStepper/archive/v0.8.2.zip
```

Comment:
We should use commit hash references since PIO isn't smart enough to pick up on moved tags. It'll cache the tag once with the old commit and then you're stuck on that until you purge your cache or there's a change to this line/version bump.